src/Call/Oneclick/InitPaymentRequest.php

```diff
@@ -1,14 +1,17 @@
 <?php declare(strict_types = 1);
 
-namespace SlevomatCsobGateway\Call;
+namespace SlevomatCsobGateway\Call\Oneclick;
 
 use DateTimeImmutable;
 use SlevomatCsobGateway\Api\ApiClient;
+use SlevomatCsobGateway\Call\PaymentResponse;
+use SlevomatCsobGateway\Call\PaymentStatus;
+use SlevomatCsobGateway\Call\ResultCode;
+use SlevomatCsobGateway\Cart;
 use SlevomatCsobGateway\Crypto\SignatureDataFormatter;
-use SlevomatCsobGateway\Currency;
 use SlevomatCsobGateway\Validator;
 
-class RecurrentPaymentRequest
+class InitPaymentRequest
```

Comment:
Rovnou bych to pojmenoval `OneclickPaymentInitRequest`, aspon se to bude jmenovat stejně jako metoda v RequestFactory a nebudeš to muset aliasovat